docs/source/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'hydra-python-agent'
+copyright = '2021, Ashish Rao'
+author = 'Ashish Rao'
```

Comment:
author to Hydra Ecosystem, maybe?

Reply:
Sure that would be better!

Reply:
Everything contributed to Hydra Ecosystem is under MIT Licence and copyright has to be assigned to "Hydra Ecosystem"